src/azul/__init__.py

```diff
@@ -1,4 +1,6 @@
 import functools
+# bind local name as pjson to avoid import conflicts with azul.json
+import json as pjson
```

Comment:
Is renaming this module required? Many other files in this directory are able to `import json` without the need to rename.

Reply:
It was required in order to get tests to pass, I'll bring it up in parking lot to see if there is a preferred way to get around the import problem.